docs/_static/simulation_models.csv

```diff
@@ -0,0 +1,28 @@
+Name; Math. symbol;Unit;Common values
+**Variables**;;;
+mass flow; :math:`\dot{m}`; :math:`kg/s`;
+mean flow velocity; :math:`v`; :math:`m/s`; :math:`1-3 m/s`
+pressure; :math:`p`; :math:`Pa` | :math:`bar`; Nominal pressures PN16 or PN25
+pressure difference; :math:`\Delta p`; :math:`Pa` | :math:`bar`; max. :math:`300\,Pa/m`
+pump power; :math:`P_{el,pump}`; :math:`W`
+temperature; :math:`T`; :math:`^\circ C`; Inlet pipe: :math:`70 - 130 ^\circ C`, Return pipe: :math:`50 - 70 ^\circ C`
+ambient temperature; :math:`T_{env}`; :math:`^\circ C`; :math:`10 ^\circ C`
+heat flow; :math:`\dot{Q}`; :math:`W`;
+**Water properties**;;;
+density; :math:`\rho`; :math:`kg/m^3`; :math:`971.78\ kg/m^3` at :math:`80^\circ C`
+spec. heat capacity; :math:`c`; :math:`J/(kg K)`; :math:`4190\,J/(kgK)` at :math:`80^\circ C` and PN16 or PN25 
+dynamic viscosity; :math:`\mu`; :math:`kg/(m \cdot s)`; :math:`0.00035\,kg/(m \cdot s)` at :math:`80^\circ C` and PN16 or PN25 
+darcy friction factor; :math:`\lambda`;--;
+Reynolds number; :math:`Re`;--;
+**Parameters**;;;
+pipe's length; :math:`L`; :math:`m`;
+pipe's inner diameter; :math:`D`; :math:`mm`; Nominal diameters DN25 - DN250
+localized pressure loss coefficient; :math:`\zeta`;--; :math:`\zeta_{tee} = 2`, :math:`\zeta_{valve} = 3.3` (for diameter of :math:`50\,mm`)
```

Comment:
```suggestion
localized pressure loss coefficient; :math:`\zeta`;--; :math:`\zeta_{tee,separating} = 2`, :math:`\zeta_{tee,uniting} = 0.75`, :math:`\zeta_{valve} = 3.3` (for diameter of :math:`50\,mm`)
```

Does it still make sense to keep the localised pressure loss coefficient in the table since we are not using it in the example? (We have different diameters in the example.)

Reply:
We use these values in the example. Should they be adapted to the diameter?

Reply:
Unfortunately the reference (VDI Wärmeatlas) only provides values of angle valves with the following diameters: 50, 100, 200, 300 and 400 mm. Plus the change of the values is not linear. For the DIN valve they only give a value range: 4 - 9.